InfiniteBlanK3T(2).CodingTracker/MainCodingTracker/UserInput.cs

```diff
@@ -0,0 +1,274 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Threading.Tasks.Sources;
+using static System.Collections.Specialized.BitVector32;
+
+namespace CodingTracker
+{
+    public class UserInput
+    {
+                
```

Comment:
Remove unnecessary empty line 13. you want one empty line between methods but no empty lines after opening bracket or before closing brackets.